src/options.lua

```diff
@@ -0,0 +1,96 @@
+--[[
+    options.lua
+
+    game options interface
+--]]
+
+local options = {
+    default_path = 'assets/options',
+    loaded_path = nil,
+
+    -- default values
+    values = {
+        fullscreen = false,
+    },
+}
+
+function options.import(path)
+    -- use default path if not provided
+    if path == nil then
+        path = options.default_path
+    end
```

Comment:
I'm assuming there isn't any intended method of setting the path right now?

Reply:
It's configurable at load-time, perhaps we could add a command-line argument for this